source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 
-Each of the searches must report 1 Searches_Regular in the DR_D1 Standard View.
+The auditor MUST run 100 searches on the platform.
 
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
 
-Audit Test D1-2: Total_Item_Requests
-....................................
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
 
-The auditor must make 100 requests on a subset of available unique items.
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time.
 
-This must result in 100 Total_Item_Requests reported in the DR_D1 Standard View.
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
 
-Multiple paths should be used to make the requests. When possible, 50% of items requested should be via browsing the platform and 50% via searching the platform.
 
-If browsing to items or accessing items via searching is not possible, then 100% of requested items can be accessed via the only available option.
+E.2.2.2 Platform Investigations and Requests
+''''''''''''''''''''''''''''''''''''''''''''
 
+Where possible, 50% of items requested SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items can be requested via the only available option.
 
-Audit Test D1-3: Total_Item_Requests - 30-second filters
-........................................................
+**Option 1**: Platforms with mixtures of Data_Types including Data_Type=Book.
 
-The audit test consists of making an Item_Request twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Request must be recorded. If the two clicks occur with more than 30 seconds between them, then 2 Total_Item_Requests must be counted.The auditor must carry out a total of 30 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
+The auditor MUST make a total of 100 requests
 
-* “Inside” tests (The 2 requests are made to the same item, and the second request is made within 30 seconds of the first).
-* “Outside” tests (The 2 requests are made to the same item, and the second request is made more than 30 seconds after the first).
+* 50 requests against different items with Data_Type other than Book.
+* 10 requests against different items within each of 5 books, for 50 total requests.
 
-The auditor must carry out 15 **inside** tests.
+If a platform only delivers book content as a whole-book download (i.e. without Book_Segments), then each request SHOULD be for a separate book.
 
-This must result in:
+The auditor activity MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 5 Unique_Title_Requests (1 per book).
```

Comment:
This is wrong for whole book downloads, since every Book_Segment is counted as a Total/Unique_Item_Investigation and Request.